odom_republisher_node:
  ros__parameters:
    odom_frame: "odom"
    base_frame: "base_link"
    camera_frame: "zed_camera_link"
    broadcast_tf: true

    use_landmark_fusion: false

    keyframe_dist_threshold: 0.3
    keyframe_yaw_threshold: 0.087

    prior_xy_noise: 0.1
    prior_yaw_noise: 0.05

    isam2_relinearize_threshold: 0.1
    isam2_relinearize_skip: 10
